src/librustdoc/html/static/css/rustdoc.css

```diff
@@ -974,7 +977,10 @@ table,
 	max-width: 100%;
 	/* contents can overflow because of max-width limit, then show ellipsis */
 	text-overflow: ellipsis;
-	border: 1px solid;
+	/* Without the `!important`, the border-color is ignored for `<select>`...
+	   It cannot be in the group above because `.search-input` has a different border color on
+	   hover. */
+	border: 1px solid var(--border-color) !important;
```

Comment:
I don't understand this comment. I think `border: 1px solid var(--border-color)` should work. 

Are you familiar with how [CSS Specificity works](https://developer.mozilla.org/en-US/docs/Web/CSS/Specificity)? Most instances of `!important` can be better solved by applying the principles of specificity.

Also, when you say "cannot be in the group above", (a) you should say "in the rule above" or "in the ruleset above" (https://developer.mozilla.org/en-US/docs/Web/CSS/Syntax#css_rulesets). Using precise language helps clarify. Also: are you talking about the ruleset with the selector `#crate-search-div`? I don't see how that one is relevant.